content/pandas/concepts/dataframe/terms/replace/replace.md

```diff
@@ -0,0 +1,90 @@
+---
+Title: '.replace()'
+Description: 'Returns DataFrame Object after values within DataFrame have been changed.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Data Structures'
+  - 'Dynamic Programming'
+  - 'Pandas'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/data-science'
+---
+
+The `.replace()` function returns a new [`DataFrame`](https://www.codecademy.com/resources/docs/pandas/dataframe) object with specified values replaced with another specified value. The original `DataFrame` object, used to call the method, remains unchanged unless explicitly declared.
+
+## Syntax
+
+```pseudo
+df = dataframe.replace(value_to_replace, new_value, inplace=True)
+```
+
+`dataframe` is the `DataFrame` with the source data and `value_to_replace` is the value you are targeting within the `DataFrame`. `new_value` is the value you want to have in place of the original value. `inplace` is `False` by default meaning that your original `DataFrame` values will not be replaced unless `inplace` is explicitly declared to `True` within the parameters.
```

Comment:
```suggestion
`dataframe` is the `DataFrame` with the source data and `value_to_replace` is the value being targeted within the `DataFrame`. `new_value` is the value used to replace the original value. `inplace` is `False` by default. The original `DataFrame` values will not be replaced unless `inplace` is explicitly declared to `True` within the parameters.
```

Reply:
For docs we generally refrain from using you/your.

Reply:
Thank you. I'll keep that in mind for all future contributions.